packages/room/README.md

```diff
@@ -0,0 +1,210 @@
+# Room Package
+
+This package is used to work with [inLive Hub and Room API services](https://hub.inlive.app/apidocs/index.html).
+
+## Usage
+
+To use this package, you need to import a `Room` module and initialize it in a global scope where you can export and import the room object returned everywhere in your application.
+
+```js
+import { Room } from '@inlivedev/inlive-js-sdk';
+// Or if you prefer to load only the room module
+import { Room } from '@inlivedev/inlive-js-sdk/dist/room.js';
+
+const room = Room()
+```
+
+### Room object
+
+The room object is the object created when the `Room` module is initialized. It consists methods that relates to the room scope. Some methods that directly interact to the room API can run on both client and server sides.
+
+#### Sample Code
+
+```js
+// create a new room
+const newRoom = await room.createRoom('a new room');
```

Comment:
@faiq-naufal is this support to create room with a custom ID?

Reply:
Mas @tyohan , this is already updated with this commit [3809023](https://github.com/inlivedev/inlive-js-sdk/pull/68/commits/3809023d41d4265abf4d721cec22f5ba2dadb294)